__tests__/nc-games.test.js

```diff
@@ -271,6 +271,59 @@ describe("/api/reviews/:review_id/comments", () => {
             })
         })
     })
+
+    describe("POST", () => {
+        test("Status 201: When POST is used on the ./comments endpoint and passed an object with a username and body, returns the posted comment.", () => {
+            return request(app)
+            .post("/api/reviews/12/comments")
+            .send({username: 'bainesface', body: 'Scythe is Lythe'})
+            .expect(201)
+            .then(({body}) => {
+                const { postedComment } = body;
+                expect(postedComment).toEqual({
+                    comment_id: 7,
+                    body: 'Scythe is Lythe',
+                    review_id: 12,
+                    author: 'bainesface',
+                    votes: 0,
+                    created_at: expect.any(String)
+                })
+            })
+        })
+
+        test("Status 400: When an incorrect data type is input, return status code 400 and 'Invalid review ID type'", () => {
+            return request(app)
+            .post("/api/reviews/twelve/comments")
+            .expect(400)
+            .send({username: 'bainesface', body: 'Scythe is Lythe'})
+            .then(({ body }) => {
+                const { msg } = body;
+                expect(msg).toBe('Invalid review ID type!')
+            })
+        })
+
+        test("Status 404: When a username that doesn't exist, return status code 404 and 'Username doesn't exist'", () => {
+            return request(app)
+            .post("/api/reviews/10/comments")
+            .send({username: 'LeeNC', body:"Didn't like this version, so I'm ditching it for a newer version."})
+            .expect(404)
+            .then(({ body }) => {
+                const { msg } = body;
+                expect(msg).toBe("Username doesn't exist");
+            })
+        })
+
+        test("Status 404: When a valid ID is searched but doesn't exist, return status code 404 and 'Review ID does not exist'", () => {
+            return request(app)
+            .post("/api/reviews/99/comments")
+            .expect(404)
+            .send({username: 'bainesface', body: 'Red Balloons is the best horror game. IT rocks.'})
+            .then(({ body }) => {
+                const { msg } = body;
+                expect(msg).toBe('Review ID does not exist!')
+            })
+        })
```

Comment:
Could be worth adding a test to check that we send a 400 bad request if `body` or `username` properties are missing as we do need it to post a comment

Reply:
Good point, will update with 2 more tests now. Thanks!